crowdin.yml

```
@@ -0,0 +1,11 @@
+---
+files:
+  -
+    source: "/google-customer-reviews/public/metadata/messages/pt-BR.json"
+    translation: "/google-customer-reviews/public/metadata/messages/%locale%.json"
+    type: "json"
+  -
+    source: "/google-customer-reviews/public/metadata/licenses/pt-BR.md"
+    translation: "/google-customer-reviews/public/metadata/licenses/%locale%.md"
```

I think we can remove this. I'm basing on this commit:
https://github.com/vtex-apps/tawk-to/commit/b36277f105328aea4521e016ded600e4eda88df4

```suggestion
    source: "/public/metadata/messages/pt-BR.json"
    translation: "/public/metadata/messages/%locale%.json"
    type: "json"
  -
    source: "/public/metadata/licenses/pt-BR.md"
    translation: "/public/metadata/licenses/%locale%.md"
```